test/performance_test/CMakeLists.txt

```
@@ -0,0 +1,47 @@
+# NOTE: We include header files to make them visible to IDEs.
+add_executable(performance_test
+    main.cpp
+    ../../src/sequential/sequential.cpp
+    ../../src/sequential/sequential.hpp
```

```suggestion
    ../../src/parallel/parallel.cpp
    ../../src/sequential/sequential.cpp
    ../../src/traces/traces.cpp

    ../../src/parallel/parallel.hpp
    ../../src/sequential/sequential.hpp
    ../../src/traces/traces.hpp
```

Missing a few source files but this should hopefully be fixed with the library overhaul.